### DISCUSSION TITLE
i3 4.20 released

### DISCUSSION THREAD

**Discussion by stapelberg** [i3/i3#4606] [Announcements]
See https://i3wm.org/downloads/RELEASE-NOTES-4.20.txt for the release notes

**Comment by poirier** [comment]
FYI, I'm not yet seeing 4.20 on the Ubuntu repo (http://debian.sur5r.net/i3/). 

It's not urgent or anything, but since it's been 8 days since the announcement I thought I'd mention it, just in case an automatic build system somewhere is hung or something.

(Detail: after "sudo apt-get update -qq", "sudo apt-get install i3-wm" output includes "i3-wm is already the newest version (4.19.1-1~~focal1)." and no new version is installed.)

**Reply by stapelberg** [reply]
@sur5r hand-maintains this repository, so there’s no stuck automation or anything :)

**Comment by sur5r** [comment]
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=998114 currently prevents i3 4.20 from entering Debian proper. Since my repo uses Debian as a starting point, it will only update once 4.20 is in Debian sid.

**Reply by Adaephon-GH** [reply]
FYI, the bug seems to have been fixed and there are now packages for i3 4.20 in [Debian sid](https://packages.debian.org/sid/i3-wm).

**Comment by poirier** [comment]
Thanks! I'll keep an eye out for 4.20 (or 4.20.1) in the Ubuntu repository.

**Comment by poirier** [comment]
FYI all, the Ubuntu repository appears to have been updated. I was able to install 4.20.1. Thanks all concerned!

**Reply by Lu-ni** [reply]
Can you explain how you get it ? 
I can't update to 4.20. https://www.ubuntuupdates.org/pm/i3-wm, It's stil at 4.19

**Comment by poirier** [comment]
I'm using the Ubuntu repository described at https://i3wm.org/docs/repositories.html, not the official Ubuntu repositories.
